crates/sui-indexer-alt-jsonrpc/src/lib.rs

```diff
@@ -43,6 +47,10 @@ pub struct RpcArgs {
     /// The maximum number of concurrent connections to accept.
     #[clap(long, default_value_t = Self::default().max_rpc_connections)]
     pub max_rpc_connections: u32,
+
+    /// The URL of the fullnode RPC we connect to for executing transactions.
+    #[clap(long, default_value_t = Self::default().fullnode_rpc_url)]
+    pub fullnode_rpc_url: String,
```

Comment:
Let's use `Url` for this, so we get some input validation. The type implements `AsRef<str>` so it can be passed to `HttpClientBuilder::build`.